src/nxt_h1proto.c

```diff
@@ -90,6 +90,8 @@ static void nxt_h1p_peer_connect(nxt_task_t *task, nxt_http_peer_t *peer);
 static void nxt_h1p_peer_connected(nxt_task_t *task, void *obj, void *data);
 static void nxt_h1p_peer_refused(nxt_task_t *task, void *obj, void *data);
 static void nxt_h1p_peer_header_send(nxt_task_t *task, nxt_http_peer_t *peer);
+static nxt_int_t nxt_h1p_peer_request_target(nxt_http_request_t *r,
+    nxt_str_t *target);
```

Comment:
This change seems unrelated to the rest. I.e I don't see you adding this function for example.

Does adding this get rid of some warning introduced by this commit?

Reply:
Sorry, I don't understand what you mean.
`nxt_h1p_peer_request_target()` is called in `nxt_h1p_peer_header_send()`.

Reply:
Ah, I see it.

<rant>
If ordered our functions correctly we wouldn't need all these prototypes...
</rant>

Reply:
Hmm, the other way is to put the function definition before the call, both the two ways are fine.

Reply:
> Hmm, the other way is to put the function definition before the call, both the two ways are fine.

If you mean this, then yes...

```c
static void func_c(void)
{
        /* ... */
}

static void func_b(void)
{
        func_c();
}

static void func_a(void)
{
        func_b();
}

int main(void)
{
        func_a();

        return 0;
}
```

Following that pattern you rarely need to provide prototypes as the compiler will see the function definition _before_ it's used.